modules/tools/AppFacCLI/README.md

```diff
@@ -0,0 +1,19 @@
+# AppFacCLI
+Google Summer of Code project to create a CLI tool for App Factory
+
+# Clone and checkout the tool
+
+First clone the repository : git clone git@github.com:Dilhasha/AppFacCLI.git
```

Comment:
Please update the readme, also improve it by providing sample CLI commands, available commands etc